### DISCUSSION TITLE
GitHub student pack

### DISCUSSION THREAD

**Discussion by Deepak9912** [community/community#41075] [GitHub Education]
### Select Topic Area

Question

### Body

Hi, Can I please have access to GitHub student pack please, so that I can apply for Heroku student credits. I am studying full stack development in code institute.

**Comment by samus-aran** [comment]
Hi @Deepak9912 👋 

We're happy you're here! Check out the [Student Developer Pack FAQ](https://github.com/orgs/community/discussions/17814) to help you apply to GitHub Global Campus.

**Comment by Kel-VinTech** [comment]
You have to apply for it and upload the required documents

**Comment by BrownFly** [comment]
好的，我会尽快给您答复。
